### DISCUSSION TITLE
Reducing event emission time

### DISCUSSION THREAD

**Discussion by farbodg** [keycloak/keycloak#26569] [Q&A]
Hello, I have noticed that it takes about 10 seconds for the `react-keycloak` library to emit the `onInitError` event. Is there anyway to reduce this?

**Comment by jonkoops** [comment]
We don't maintain `react-keycloak`, in fact the library is not maintained [at all anymore](https://github.com/react-keycloak/react-keycloak?tab=readme-ov-file#warning-deprecated-and-unmaintained). I'd recommend that you look for another library.
